test/byContract/IexecPoco/04_finalize.test.ts

```diff
@@ -558,6 +607,58 @@ describe('IexecPoco2#finalize', async () => {
         );
     });
 
+    it('Should finalize task when result callback is invalid', async () => {
```

Comment:
```suggestion
    it('Should finalize task when callback address is non-EIP1154 contract', async () => {
```
And move test closer to the EOA one?